### PR TITLE
[34475] Support emoji 13.0

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

Support file for the changes in https://github.com/mattermost/mattermost-webapp/pull/8220

#### Ticket Link
[MM-34475](https://mattermost.atlassian.net/browse/MM-34475)

#### Release Note
```release-note
support emoji standard v13.0
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
😍👌❤️😀😃😄😁

**Review by hmhealey — APPROVED** [review]

File: model/emoji_data.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
...: "1f4ef", "swimming_woman_dark_skin_tone": "1f3ca-1f3ff-200d-2640-fe0f", "raised_hand_with_fingers_splayed": "1f590", "ok_man": "1f646-200d-2642-fe0f", "new": "1f195", "male_detective_medium_dark_skin_tone": "1f575-1f3fe-200d-2640-fe0f", "computer_mouse": "1f5b1", "hourglass_flowing_sand": "23f3", "bahrain": "1f1e7-1f1ed", "djibouti": "1f1e9-1f1ef", "zimbabwe": "1f1ff-1f1fc", "swimming_man_light_skin_tone": "1f3ca-1f3fb-200d-2640-fe0f", "interrobang": "2049-fe0f", "clock8": "1f557", "pancakes": "1f95e", "thermometer": "1f321", "label": "1f3f7", "denmark": "1f1e9-1f1f0", "raising_hand_man_medium_light_skin_tone": "1f64b-1f3fc-200d-2642-fe0f", "frowning_man_medium_dark_skin_tone": "1f64d-1f3fe-200d-2642-fe0f", "point_right": "1f449", "guitar": "1f3b8", "family_woman_woman_girl_medium_skin_tone": "1f469-1f3fd", "woman_juggling_medium_skin_tone": "1f939-1f3fd-200d-2640-fe0f", "man_health_worker": "1f468-200d-2695-fe0f", "stew": "1f372", "surfing_man": "1f3c4", "twisted_rightwards_arrows": "1f500", "timor_leste": "1f1f9-1f1f1", "weight_lifting_woman": "1f3cb-fe0f-200d-2640-fe0f", "amphora": "1f3fa", "heart": "2764-fe0f", "bowing_man_medium_dark_skin_tone": "1f647-1f3fe-200d-2640-fe0f"}
```
Comment (jazzzz) [inline model/emoji_data.go:6]: Some custom emojis like rage* disappeared with this change, is it normal?

Comment (Willyfrog) [reply model/emoji_data.go:6]: > Some custom emojis like rage* disappeared with this change, is it normal?

yes, it was intended, also the original library stopped providing support as well. For getting them back you can go to this [previous release of the previous library](https://github.com/github/gemoji/tree/v2.1.0/images/emoji) and add them back as custom emojis

Comment (jazzzz) [reply model/emoji_data.go:6]: OK, I did not see this change in the release notes so I was not sure. Thanks for the link!

**Pull request merged**